charts/shoot-addons/charts/nginx-ingress/templates/clusterrole.yaml

```diff
@@ -12,24 +12,16 @@ rules:
   - apiGroups:
       - ""
     resources:
+{{- if semverCompare "< 1.20-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
What do you think about passing and comparing the ingress controller version instead of comparing the K8s version? This is a little bit confusing, IMO.

Reply:
Good idea, I thought about the same, but I don't know the ingress-controller version. Due to the image vector overwrite some sha can be injected, and then the comparison does not work anymore. :(

Reply:
Argh .... ok 😞